dd-sdk-android/src/test/kotlin/com/datadog/android/log/LoggerTest.kt

```diff
@@ -833,4 +837,114 @@ internal class LoggerTest {
     }
 
     // endregion
+
+    // region Multi Thread Access
+
+    @Test
+    fun `adding tags or attributes is thread safe`(forge: Forge) {
```

Comment:
This test method is huge. Can you break it in two, one to test tags, and one to test attributes. 
Maybe try to make the code a bit more readable, it's hard to follow your logic here.

Reply:
Agree...good idea to break it in 2 for tags and attributes.